core/src/test/java/com/google/cloud/sql/core/CloudSqlInstanceConcurrencyTest.java

```diff
@@ -52,91 +51,7 @@ public void initialize(HttpRequest var1) throws IOException {
     }
   }
 
-  @Test(timeout = 20000)
-  public void testThatForceRefreshBalksWhenARefreshIsInProgress() throws Exception {
```

Comment:
I think this test was built around the old refresh behavior -- where current would block on the next refresh. I need to double check that we have good coverage for balking before I ask for another review.